lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb

```diff
@@ -114,21 +118,34 @@ def portmap
 
   def envmap
     # TODO: allow commas inside quotes
-    @fields['environment_variables'].split(',').map {|x| "-e #{x}" }.join(' ') if @fields.key?('environment_variables')
+    @envs.split(',').map {|x| "-e #{x}" }.join(' ')
   end
 
   def linkmap
-    @fields['links'].split(',').map {|x| "--link #{x}" }.join(' ') if @fields.key?('links')
+    @links.split(',').map {|x| "--link #{x}" }.join(' ')
   end
 
   def volmap
-    @fields['volumes'].split(',').map {|x| "-v #{x}" }.join(' ') if @fields.key?('volumes')
+    @vols.split(',').map {|x| "-v #{x}" }.join(' ')
   end
 
   def container_name(image_name)
     "--name coopr-#{image_name.split('/').last}"
   end
 
+  def setup_host_volumes(volumes)
+    volumes.split(',').each do |vol|
+      # Does the host-side exist, if so, do nothing
+      begin
+        remote_command("test -d #{vol.split(':').first}")
+        continue
```

Comment:
i think you want ``next``, or potentially nothing at all since there's no further code to skip